trash.conf

```diff
@@ -17,7 +17,7 @@ github.com/docker/docker bf16bd9dcfc3c9fafb7eb7b39ae7ef7abf1ae7f1 https://github
 github.com/docker/engine-api v0.3.3
 github.com/docker/go-connections v0.2.0
 github.com/docker/go-units 651fc226e7441360384da338d0fd37f2440ffbe3
-github.com/docker/libcompose 09abf78a849d3cb81a4c17b48d0b68af713a80a5 https://github.com/rancher/libcompose.git
+github.com/docker/libcompose b9f88a071f78d49293a8f31ae0d2e339085fd0d7 https://github.com/rancher/libcompose.git
```

Comment:
We're working off of [this branch](https://github.com/rancher/libcompose/tree/os) in our fork. I backported my commit from upstream to this branch.